spec/configuration_spec.rb

```diff
@@ -1,3 +1,4 @@
+require 'spec_helper'
```

Comment:
I believe this line is unnecessary. The tests run well without it. Can you double check?

Reply:
Sure, I will check both require statements.

Reply:
Once I remove this spec_helper the configuration file is not considered on the coverage report, maybe due to different folder hierarchy. 🤔